mlrun/feature_store/feature_set.py

```diff
@@ -812,6 +830,17 @@ def to_dataframe(
             if self.spec.timestamp_key and self.spec.timestamp_key not in entities:
                 columns = [self.spec.timestamp_key] + columns
             columns = entities + columns
+
+        if self.spec.passthrough:
+            if not self.spec.source:
+                raise mlrun.errors.MLRunNotFoundError(
+                    (
+                        "passthrough feature-set with no source binded. "
+                        "this happens at ingest, was it executed on this feature-set?"
```

Comment:
I think this bit is just adding to the confusion since this shouldn't happen anyway.

Reply:
fixed